components/remote/os.go

```diff
@@ -0,0 +1,46 @@
+package remote
+
+import (
+	"github.com/DataDog/test-infra-definitions/common/config"
+	"github.com/DataDog/test-infra-definitions/components/command"
+	"github.com/DataDog/test-infra-definitions/components/os"
+
+	"github.com/pulumi/pulumi-command/sdk/go/command/remote"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func MakeHost(e config.CommonEnvironment, conn remote.ConnectionOutput, osDesc os.Descriptor, osUser string, readyFunc command.ReadyFunc, host *Host) error {
```

Comment:
No, it's a function that fills a `Host` object from some generic input to avoid mistakes and repetitive code in all VM creation. We do not create anything in there. `InitHost` would work as well, but perhaps the intent is a bit less clear

Reply:
Reading it I thought it was creating a host. It was weird that it was taking a host pointer and filled it. Maybe adding a description is enough.